util/import.ts

```diff
@@ -1,11 +1,50 @@
 import { TestCaseInfo } from '../state/atoms/patientTestCase';
 
-export function bundleToTestCase(bundle: fhir4.Bundle): TestCaseInfo {
+export function bundleToTestCase(bundle: fhir4.Bundle, populationGroupCodes: string[]): TestCaseInfo {
   if (!bundle.entry || bundle.entry.length === 0) {
     throw new Error('Bundle has no entries');
   }
 
   const patientEntry = bundle.entry.find(e => e.resource?.resourceType === 'Patient');
+  let desiredPopulations: string[] = [];
+  const testCaseMeasureReportArr = bundle.entry.filter(isTestCaseMeasureReport);
+  if (testCaseMeasureReportArr.length > 1) {
+    // TODO: Once we have import errors persist on page, replace this!!!
+    throw new Error(
+      `Expected 1 or 0 test case measure reports in bundle, but found ${testCaseMeasureReportArr.length}`
+    );
+  }
+  if (testCaseMeasureReportArr.length === 1) {
+    // use reduce here to retrieve population codes and filter out potential undefined codes
+    let retrievedPops = (testCaseMeasureReportArr[0].resource as fhir4.MeasureReport)?.group?.[0]?.population?.reduce(
```

Comment:
Something that probably warrants a larger discussion is how to handle having multiple groups. For the most part, I think we just look at the first group in the array, but with the advancement of ratio measure support and other issues brought up by SB, I think we should make sure we are consistent with whether we look at the first group versus loop over all groups.

Here, for example, we look at the first group, but in `getMeasurePopulations()` (from `util/MeasurePopulations.ts`) we loop over all the groups. Should we change the logic in `getMeasurePopulations()` to only look at the first group since we are only looking at the first group here?